src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc

```diff
@@ -110,8 +124,7 @@ TEST_F(AlertManeuverRequestTest, Run_ProcessingResult_UNSUCCESS) {
   const mobile_apis::Result::eType kProcessingResult =
       mobile_apis::Result::ABORTED;
 
-  EXPECT_CALL(*(am::MockMessageHelper::message_helper_mock()),
-              ProcessSoftButtons(_, _, _, _))
+  EXPECT_CALL(mock_message_helper_, ProcessSoftButtons(_, _, _, _))
```

Comment:
@indlu , in this expect call you can check first arg.
Please, check other test cases